lib/tests/streamlit/data_mocks.py

```diff
@@ -388,8 +400,6 @@ def __str__(self):
             for i in range(3)
         ]
         + [None],
-        "period[D]": [(pd.Period(random_date().date(), freq="D")) for _ in range(3)]
```

Comment:
`period[H]` (see the line above) is now also supported and can be removed here or?